test/test_patch.rb

```diff
@@ -104,4 +104,10 @@ def test_merges_similar_ids_but_different_signs
       assert_equal(Zold::Amount.new(zld: 2.0), first.balance)
     end
   end
+
+  def test_not_allow_amount_exceed_max
+    # @todo Write test for Patch#join when joined wallet
```

Comment:
@liligga run [pdd](https://github.com/yegor256/pdd#how-to-format) and check the formatting of this puzzle.